cmd/syncthing/main.go

```diff
@@ -728,6 +708,34 @@ func syncthingMain() {
 	cachedDiscovery := discover.NewCachingMux()
 	mainSvc.Add(cachedDiscovery)
 
+	// GUI
+
+	setupGUI(mainSvc, cfg, m, apiSub, cachedDiscovery, relaySvc, errors, systemLog)
+
+	// Initialize Folders
```

Comment:
Also potential for posting a new config before we initialize the folders, perhaps.